unit-tests-java8/src/test/java/org/eclipse/collections/test/map/mutable/MutableMapTestCase.java

```diff
@@ -53,4 +61,36 @@ default void Iterable_remove()
                 this.getExpectedFiltered(3, 3, 3, 2, 1),
                 this.getExpectedFiltered(3, 3, 2, 2, 1)));
     }
+
+    @Test
+    default void Iterable_computeIfPresent()
```

Comment:
This test should move up to MutableMapIterableTestCase, and be named Map_computeIfPresent()